server.go

```diff
@@ -1554,9 +1554,16 @@ func (s *Server) applyCreateMeasurementsIfNotExists(m *messaging.Message) error
 		return ErrDatabaseNotFound
 	}
 
+	measurements := make([]string, 0, len(c.Measurements))
```

Comment:
Please add a comment explaining that is purely for consistency of operation, since nothing in our design *requires* this to happen.